examples/03_connectivity/plot_confounds_removal.py

```diff
@@ -0,0 +1,167 @@
+"""
+Carefully defining the confounds
+================================
+
+This example shows the importance of confounds removal for functional
+connectivity.
+
+"""
+########################################
+# We use one ADHD subject.
+from nilearn import datasets
+adhd = datasets.fetch_adhd(n_subjects=1)
+func_filename = adhd.func[0]
+
+######################################
+# Friston 24 motion parameters
+# -------------------------------------
+# They include the 6 motion parameters of the current volume
+# and the preceding volume, plus each of these values squared.
+
+# Read the 6 motion parameters from the confounds file
+import numpy as np
+csv_confounds = np.recfromcsv(adhd.confounds[0], delimiter='\t')
+current_volume_motion = [
+    csv_confounds[name] for name in ('motionx', 'motiony', 'motionz',
+                                     'motionpitch', 'motionroll', 'motionyaw')]
```

Comment:
You should be able to index it directly `csv_confounds[['motionx', 'motiony', 'motionz', 'motionpitch', 'motionroll', 'motionyaw']]`